### PR TITLE
Fix Pachyderm links

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

**Comment by bdattoma** [comment]
Although the new links look good, I would wait to receive a confirmation about the desired links to use, because the same wrong ones are present in RHODS: RHODS-7424.

**Comment by diegolovison** [comment]
https://github.com/red-hat-data-services/odh-deployer/pull/310

**Review by jgarciao — APPROVED** [review]
lgtm

**Review by bdattoma — DISMISSED** [review]

**Comment by bdattoma** [comment]
please run the PR test in Jenkins

File: ods_ci/tests/Resources/Files/AppsInfoDictionary.json [hunk 1]
```diff
@@ -244,7 +244,7 @@
                    },
                    "8": {
                          "text": "tutorial",
-                         "url": "https://docs.pachyderm.com/latest/getting_started/beginner_tutorial/",
+                         "url": "https://docs.pachyderm.com/latest/getting-started/beginner-tutorial/",
                          "matching": ""
                    },
                    "9": {
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Files/AppsInfoDictionary.json:250]: these changes are not required, only the json file with _latest suffix should be changed. This one is used for retrocompatibility with previous version of RHODS

**Review by bdattoma — CHANGES_REQUESTED** [review]

**Review by bdattoma — APPROVED** [review]

**Comment by diegolovison** [comment]
It worked https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1716/robot/report/files/test_report.html
I removed later the `ProductBug`

**Review by bdattoma — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Pull request merged**